test/e2e/handler/examples_test.go

```diff
@@ -49,6 +49,12 @@ var _ = Describe("[user-guide] Examples", func() {
 			policyName: "linux-bridge",
 			ifaceNames: []string{"br1"},
 		},
+		exampleSpec{
+			name:       "Linux bridge absent",
```

Comment:
Detach bridge port and restore its configuration

I think that would speak more for the intent